lib/demos/material/divider_demo.dart

```diff
@@ -0,0 +1,74 @@
+// Copyright 2019 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/gallery_localizations.dart';
+
+// BEGIN dividerDemo
+
+class DividerDemo extends StatelessWidget {
+  const DividerDemo({Key key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final localization = GalleryLocalizations.of(context);
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(

```

Comment:
Make sure the back button doesn't show on this app bar by setting automaticallyImplyLeading to false